test/test-main.js

```diff
@@ -0,0 +1,12 @@
+var main = require("./main");
+
+exports["test main"] = function(assert) {
+  assert.pass("Unit test running!");
```

Comment:
I don't think we are going to want a different unit testing framework for privly-jetpack from all the other repositories, but I could be wrong since I am not familiar with this framework. Feel free to disagree.